projects/tuist/features/precompiled.feature

```diff
@@ -33,21 +33,21 @@ Feature: A set of tests that run with pre-compiled binaries that are only compat
     Then the product 'App.app' with destination 'Debug-iphonesimulator' contains the framework 'Framework1' with architecture 'x86_64'
     Then the product 'App.app' with destination 'Debug-iphonesimulator' contains the framework 'Framework2' without architecture 'arm64'
     Then the product 'App.app' with destination 'Debug-iphonesimulator' does not contain headers
-    Then I should be able to build for iOS the scheme AppUITests
+    Then I should be able to build for iOS the scheme App
     Then the product 'AppUITests-Runner.app' with destination 'Debug-iphonesimulator' does not contain the framework 'Framework2'
     Then I should be able to build for iOS the scheme Framework1-iOS
     Then I should be able to build for macOS the scheme Framework1-macOS
     Then I should be able to build for iOS the scheme Framework1Tests-iOS
     Then I should be able to build for macOS the scheme Framework1Tests-macOS
-    Then I should be able to build for iOS the scheme StaticFramework1Tests
+    Then I should be able to build for iOS the scheme StaticFramework1
```

Comment:
The acceptance tests were specifically after building those particular targets - I'd suggest we change the fixture to use the `.single` (or the equivalent of today's default) - rather than have it implicitly rely on the new grouping behaviour.

Reply:
The `StaticFramework1` scheme now includes the `StaticFramework1Tests` target, so we can just build that scheme, isn't it?